### PR TITLE
Feature add config files and libraries

### PR DESCRIPTION
**Pull request by borovHokage**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN dotnet restore -s https://api.nuget.org/v3/index.json
+
+COPY . ./
+RUN dotnet publish -c Release -o out
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-bullseye-slim AS base
+WORKDIR /app
+COPY --from=build /app/out .
+EXPOSE 80
+ENTRYPOINT ["dotnet", "LT.DigitalOffice.FileService.dll"]
```
Comment (MariyaSin) [inline Dockerfile]: fileServuice?

File: LT.DigitalOffice.FilterService.sln [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		Dockerfile = Dockerfile
+		..\DigitalOffice-FileService\.github\workflows\main.yml = ..\DigitalOffice-FileService\.github\workflows\main.yml
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FilterService.Broker", "src\FilterService.Broker\FilterService.Broker.csproj", "{08D880D8-21EF-479A-8604-01F278AE8C75}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FilterService.Business", "src\FilterService.Business\FilterService.Business.csproj", "{546F2430-170B-4C33-BE29-A44E7AA38DF2}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FilterService.Data", "src\FilterService.Data\FilterService.Data.csproj", "{FBF41E78-4A03-4394-B9FB-DE67D2F43E10}"
```
Comment (MariyaSin) [inline LT.DigitalOffice.FilterService.sln]: we do not have data in the service

File: src/FilterService/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	}
+
+	#endregion
+
+	public Startup(IConfiguration configuration)
+	{
+	  Configuration = configuration;
+
+	  _serviceInfoConfig = Configuration
+		  .GetSection(BaseServiceInfoConfig.SectionName)
```
Comment (MariyaSin) [inline src/FilterService/Startup.cs]: tabs

File: src/FilterService/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+		  .Get<BaseServiceInfoConfig>();
+
+	  _rabbitMqConfig = Configuration
+		  .GetSection(BaseRabbitMqConfig.SectionName)
+		  .Get<BaseRabbitMqConfig>();
+
+	  Version = "1.0.0";
```
Comment (MariyaSin) [inline src/FilterService/Startup.cs]: 1.0.0.0

File: src/FilterService/appsettings.Development.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "RabbitMQ": {
+    "Host": "localhost",
+    "Username": "guest",
+    "Password": "guest"
+  },
+  "MemoryCache": {
+    "CacheLiveInMinutes": 5
+  },
+  "Redis": {
+    "CacheLiveInMinutes": 20
```
Comment (MariyaSin) [inline src/FilterService/appsettings.Development.json]: you do not add redis there

File: src/FilterService/appsettings.Production.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "Serilog": {
+  },
+  "RabbitMQ": {
+    "Host": "lt_do_rabbitmq"
+  },
+  "EmailEngineConfig": {
```
Comment (MariyaSin) [inline src/FilterService/appsettings.Production.json]: ????

File: Dockerfile (hunk 1 above)
Comment (borovHokage) [reply Dockerfile]: fixed

File: LT.DigitalOffice.FilterService.sln (hunk 2 above)
Comment (borovHokage) [reply LT.DigitalOffice.FilterService.sln]: removed

File: src/FilterService/Startup.cs (hunk 4 above)
Comment (borovHokage) [reply src/FilterService/Startup.cs]: done

File: src/FilterService/appsettings.Development.json (hunk 5 above)
Comment (borovHokage) [reply src/FilterService/appsettings.Development.json]: done

File: src/FilterService/appsettings.Production.json (hunk 6 above)
Comment (borovHokage) [reply src/FilterService/appsettings.Production.json]: rremoved

**Review by MariyaSin — APPROVED** [review]

File: src/FilterService/Startup.cs (hunk 3 above)
Comment (borovHokage) [reply src/FilterService/Startup.cs]: alright/ thas is git's problems

File: src/FilterService/FilterService.csproj [hunk 7]
```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+	
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+	
+  <ItemGroup>
+	<PackageReference Include="AspNetCore.HealthChecks.SqlServer" Version="5.0.2" />
+	<PackageReference Include="AspNetCore.HealthChecks.UI.Client" Version="5.0.1" />
```
Comment (ndoriomedov) [inline src/FilterService/FilterService.csproj]: update to 6.0.1 or older

**Review by ndoriomedov — APPROVED** [review]

File: src/FilterService/FilterService.csproj (hunk 7 above)
Comment (borovHokage) [reply src/FilterService/FilterService.csproj]: ok

**Pull request merged**